### PR TITLE
Update on maintainerships owned by @nvlsianpu

### PR DESCRIPTION
**Pull request by carlescufi**
Since @nvlsianpu is moving on to a separate internal team at Nordic, replace him with other engineers in the corresponding files.

### CODE REVIEW

**Review by de-nordic — DISMISSED** [review]

**Review by stephanosio — DISMISSED** [review]

**Review by nordicjm — DISMISSED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -197,8 +197,8 @@
 /CMakeLists.txt                           @tejlmand @nashif
 /doc/                                     @carlescufi
 /doc/develop/tools/coccinelle.rst                @himanshujha199640 @JuliaLawall
-/doc/services/device_mgmt/smp_protocol.rst  @de-nordic
-/doc/services/device_mgmt/smp_groups/       @de-nordic
+/doc/services/device_mgmt/smp_protocol.rst  @de-nordic @nordicjm
+/doc/services/device_mgmt/smp_groups/       @de-nordic @nordicjm
```
Comment (MaureenHelm) [inline CODEOWNERS:201]: This change doesn't involve Andrzej

File: CODEOWNERS [hunk 2]
```diff
@@ -687,7 +687,7 @@
 /samples/subsys/logging/                  @nordic-krch @jakub-uC
 /samples/subsys/logging/syst/             @dcpleung
 /samples/subsys/shell/                    @jakub-uC @nordic-krch
-/samples/subsys/mgmt/mcumgr/smp_svr/      @aunsbjerg @nvlsianpu
+/samples/subsys/mgmt/mcumgr/              @de-nordic @nordicjm
```
Comment (MaureenHelm) [inline CODEOWNERS:690]: @aunsbjerg

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (carlescufi) [reply CODEOWNERS:201]: Would you like me to remove this line?

File: CODEOWNERS (hunk 2 above)
Comment (carlescufi) [reply CODEOWNERS:690]: Note that the reason we removed @aunsbjerg is that he is already present in the specific file he maintains.

**Review by nvlsianpu — DISMISSED** [review]

**Review by nvlsianpu — COMMENTED** [review]
I will still doing synchronizations of zephyr-rtos/mcuboot with mcu-tools/mcuboot.

File: CODEOWNERS (hunk 2 above)
Comment (MaureenHelm) [reply CODEOWNERS:690]: Ok, just mention this in the commit message

**Comment by Laczen** [comment]
Since maintainers are more than just a line that can be replaced in a maintainers file:

Andrzej (@nvlsianpu), I would sincerely like to thank you for the hard work you have done for zephyr. You have added support for storage, fcb, settings, ... have been the maintainer for flash for a large time and have helped zephyr to become what it is today. Being a maintainer comes with responsabilities that you have always done well. Of course we didn't always agree on the right decision to take at certain moments, but that is OK. I wish you all the best with new responsabilities and I hope you will still visit the community from time to time.

File: CODEOWNERS (hunk 1 above)
Comment (carlescufi) [reply CODEOWNERS:201]: Done, thanks!

File: CODEOWNERS (hunk 2 above)
Comment (carlescufi) [reply CODEOWNERS:690]: Done, thanks!

**Review by de-nordic — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**